alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -47,6 +48,7 @@ class embedPlaykitJsAction extends sfAction
 	private $playerConfig = null;
 	private $uiConfUpdatedAt = null;
 	private $regenerate = false;
+	private $includeSourceMap = "false";
```

Comment:
I would suggest to load this default value from config, so in case we are catastrophy mode, we can change this value fast, but in general - hold it with value **true**

Reply:
It doesn't make sense the default will be true as the entire purpose of this change is that the source map won't be returned by default.